index.html

```diff
@@ -0,0 +1,350 @@
+    <meta charset="utf-8">
+    <meta http-equiv="x-ua-compatible" content="ie=edge">
+    <title>Avengers</title>
+    <meta name="description" content="">
+    <meta name="viewport" content="width=device-width, initial-scale=1">
+
+    <link rel="apple-touch-icon" href="apple-touch-icon.png">
+
+    <link rel="stylesheet" href="/css/reset.css">
+    <link rel="stylesheet" href="/css/style.css">
+  </head>
+  <body>
+    <nav class="menu">
+      <a href="#home">Home.</a>
+      <a href="#services">Services.</a>
+      <a href="#portfolio">Portfolio.</a>
+      <a href="#about">About.</a>
+      <a href="#team">Team.</a>
+      <a href="#contact">Contact.</a>
+    </nav>
+    <header id="home" class="header-content">
+      <img class="logo circle" width="21" height="32" src="/img/tear-drop-two.svg" alt="Logo for Avengers" title="Avengers logo">
+      <h1 class="header-logo-text">Hi there! We're Avengers, <strong>creative design agency</strong> we build <strong class="curve-underline">awesome</strong> mobile and websites</h1>
+    </header>
+    <main>
+      <article id="services" class="services-container">
+        <h2 class="section-header white-color white-line">SERVICES</h2>
+        <p class="services-text">Contrary to popular belief, Lorem Ipsum is not simply random text. It has roots in a piece of classical Latin literature from 45 BC, making it over 2000 years old. Richard McClintock, a Latin professor at Hampden-Sydney College in Virginia.</p>
+        <p class="services-text small-paragraph">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+        <p class="services-text small-paragraph">Lorem Ipsum is simply dummy text of the printing and typesetting industry. Lorem Ipsum has been the industry's standard dummy text ever since the 1500s, when an unknown printer took a galley of type and scrambled it to make a type specimen book.</p>
+        <div class="services-category-container">
+          <section class="services-category">
+            <h3 class="services-header white-color" data-title="ux-design">UI &amp; UX DESIGN</h3>
+            <p class="services-text">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+          </section>
+          <section class="services-category">
+            <h3 class="services-header white-color" data-title="laboratory">LABORATORY</h3>
+            <p class="services-text">There are many variations of passages of Lorem Ipsum available, but the majority have suffered alteration in some form, by injected support by humour.</p>
+          </section>
+          <section class="services-category">
+            <h3 class="services-header white-color" data-title="development">DEVELOPMENT</h3>
```

Comment:
If left as a `section` (which I don't think it should be), this should be an `h1`. If not, it should be an `h2`.